srcpkgs/lets/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'lets'
+pkgname=lets
+version=0.0.29
+revision=1
+build_style=go
+go_import_path="github.com/lets-cli/lets"
+short_desc="CLI task runner for developers - a better alternative to make"
+maintainer="juan <juan@horlux.org>"
+license="MIT"
+homepage="https://lets-cli.org/"
+distfiles="https://$go_import_path/archive/v$version.tar.gz"
```

Comment:
Please don't use `go_import_path` here.

Reply:
fixed, although I took it from the browsh pkg